core/src/main/resources/jenkins/monitor/JavaLevelAdminMonitor/message.properties

```diff
@@ -0,0 +1,3 @@
+Recommended_Java_Version=Java 11 is the recommended version to run Jenkins on, please consider upgrading. \
```

Comment:
please suggest any wording improvements

Reply:
Maybe something like Jenkins runs better on Java 11, too? Some sort of treat.